### ISSUE TITLE
NullReferenceException: Object reference not set to an instance of an object?

### ISSUE THREAD

**Issue by krissyboo222** [bozzobrain/LethalCompanyShipMaid#21]
Been Struggling with some issues for a few hours.  I type "cleanup" in the terminal and nothing happens.  Attached is the error I'm getting (other similar errors have happened with different items), a copy of the .cfg (but in .txt form) and a link to the modpack I am trying to use this on.

Some help would be hugely appreciated!  This mod was super cool when it was working, and I'm just not sure how I broke it so badly.

https://thunderstore.io/c/lethal-company/p/LethalPolycule/Lethal_Polycule/

![0JIEPGB](https://github.com/bozzobrain/LethalCompanyShipMaid/assets/43711097/564a8792-416f-4551-b5be-679e9c5e4528)
[ShipMaidcfg.txt](https://github.com/bozzobrain/LethalCompanyShipMaid/files/14638487/ShipMaidcfg.txt)

**Comment by bozzobrain** [comment]
Is this a recurring issue? I have seen it a number of times, but usually I will only see it itermittently. Having said that it does look like it is a result of the shotgun organization within the storage closet. Can you try not having this item temporarily not organized on a shelve and see if that is the cause of the issue.

I do know that sometimes the networking manager fails to register and throws a similar issue. 

I can try to replicate the issue soon, but those suggestions may get you back up and running sooner in the meantime.

Sorry that you are having trouble with the mod!

**Comment by krissyboo222** [comment]
I received your message. I'm out of town atm and will be for a few days. I
won't be back till Sunday or Monday. When I have some downtime, I'll see if
I can set stuff up on my laptop to do more testing.

It was doing it with different objects, it wasn't just the shotgun. I
forgot what else it did it to, but after I got that error, I tried
adjusting the order of the stuff that goes in the cabinet, and the item
causing errors did change after that. Sadly, the error was persistent.
